examples/mediadevices/src/helpers.js

```diff
@@ -14,42 +15,54 @@ function getDevicesOfKind(deviceInfos, kind) {
   });
 }
 
+function switchLocalTracks(room, newLockTrack) {
+  if (room) {
+      room.localParticipant.tracks.forEach(function(trackPublication) {
+      if (trackPublication.kind === newLockTrack.kind) {
+        room.localParticipant.unpublishTrack(trackPublication.track);
+      }
+    });
+    room.localParticipant.publishTrack(newLockTrack);
+  }
+}
+
 /**
  * Apply the selected audio input device.
  * @param {string} deviceId
  * @param {HTMLAudioElement} audio
+ * @param {Room} [room] to switch tracks on.
```

Comment:
@makarandp0 `* @param {Room} [room] - The Room, if you have already joined one`?